### PR TITLE
Fix #2874 shouldn't use the words "success" and "failure" 

### LINKED ISSUE #2874: "check category failure title" and "check category success title" shouldn't use the words "success" and "failure"
These two messages say:
* Requesting category check: Success
* Requesting category check: Failed

Using the words "Success" and "Failed" in user interface messages is a bad practice. See the explanation here: https://www.mediawiki.org/wiki/Localisation#Avoid_jargon_and_slang .

They should be rephrased in a more human way. For example:
* Category check requested
* Category check request didn't work: *explanation with suggested actions*

### PR DESCRIPTION
**Pull request by isdl03**
Description (required)

Fixes #2874

What changes did you make and why?
Replaced the following three strings :
1)Login success!
You are logged in!

Wallpaper set successfully!
Wallpaper set!

Sending Thanks: Success
Thanks Sent!

Tests performed (required)

Tested on Pixel 3 with API level 28.

Need help? See https://support.google.com/android/answer/9075928

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -50,7 +50,7 @@
   <string name="signup">Sign up</string>
   <string name="logging_in_title">Logging in</string>
   <string name="logging_in_message">Please wait…</string>
-  <string name="login_success">Login success!</string>
+  <string name="login_success">You are logged in!</string>
```
Comment (sivaraam) [inline app/src/main/res/values/strings.xml:53]: I believe "Logged in!" would be shorter and equally precise here.

**Review by sivaraam — CHANGES_REQUESTED** [review]
Thanks for the PR! I've left one inline comment. Also, to completely address #2874 we would've to replace _all_ instances of the usage mentioned there. It looks like there are still some strings left even after this change. A few instances are,

https://github.com/commons-app/apps-android-commons/blob/44bf6048e970eb0e4b8d2ffac0c2e40ee547c014/app/src/main/res/values/strings.xml#L514

https://github.com/commons-app/apps-android-commons/blob/44bf6048e970eb0e4b8d2ffac0c2e40ee547c014/app/src/main/res/values/strings.xml#L516

https://github.com/commons-app/apps-android-commons/blob/44bf6048e970eb0e4b8d2ffac0c2e40ee547c014/app/src/main/res/values/strings.xml#L515

So, it would be great if you could fix those too.

Also, make sure to properly update the "Fixes #INSERT_ISSUE_NUMBER_HERE" template in the PR description to mention #2874

**Comment by arishta** [comment]
@sivaraam We have made the changes as requested. Kindly review the same.

**Comment by sivaraam** [comment]
> @sivaraam We have made the changes as requested. Kindly review the same.

Apologies for the delay. I was supposed to get around to this long ago but it slipped. Currently, I'm in lack of time due to a medical emergency in my family. I'll be sure to review when I find the time. We'll see if someone else could beat me to it, instead :)

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -48,24 +48,24 @@
...
   <string name="upload_progress_notification_title_finishing">Finishing uploading %1$s</string>
-  <string name="upload_failed_notification_title">Uploading %1$s failed</string>
+  <string name="upload_failed_notification_title">Cannot upload %1$s</string>
   <string name="upload_paused_notification_title">Uploading %1$s paused</string>
   <string name="upload_failed_notification_subtitle">Tap to view</string>
   <string name="upload_paused_notification_subtitle">Tap to view</string>
   <string name="title_activity_contributions">My Recent Uploads</string>
   <string name="contribution_state_queued">Queued</string>
-  <string name="contribution_state_failed">Failed</string>
+  <string name="contribution_state_failed">Error detected</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml:68]: I think failed fits better for this specific usage. Error detected does not indicate that upload process failed, ie. we may have an error for adding categories (so error detected) and upload may be done. So, "error detected" is not necessarily mean "upload failed".

Suggestions are welcome, I am not the most native Ensglish speaker I know.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -78,12 +78,12 @@
...
   <string name="share_caption_hint">Caption</string>
-  <string name="login_failed_network">Unable to login - network failure</string>
+  <string name="login_failed_network">Unable to login - network connection lost</string>
   <string name="login_failed_wrong_credentials">Unable to login - please check your username and password</string>
-  <string name="login_failed_throttled">Too many unsuccessful attempts. Please try again in a few minutes.</string>
+  <string name="login_failed_throttled">Too many invalid attempts. Please try again in a few minutes.</string>
   <string name="login_failed_blocked">Sorry, this user has been blocked on Commons</string>
   <string name="login_failed_2fa_needed">You must provide your two factor authentication code.</string>
-  <string name="login_failed_generic">Login failed</string>
+  <string name="login_failed_generic">Error logging in</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml:86]: Is it legal to say "Error logging in" gramatically?

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -586,9 +586,9 @@ Upload your first media by tapping on the add button.</string>
   <string name="default_description_language">Default description language</string>
   <string name="delete_helper_make_deletion_toast">Trying to nominate %1$s for deletion</string>
   <string name="delete_helper_show_deletion_title">Nominating for deletion</string>
-  <string name="delete_helper_show_deletion_title_success">Success</string>
+  <string name="delete_helper_show_deletion_title_success">Deleted</string>
   <string name="delete_helper_show_deletion_message_if">Nominated %1$s for deletion.</string>
-  <string name="delete_helper_show_deletion_title_failed">Failed</string>
+  <string name="delete_helper_show_deletion_title_failed">Title could not be deleted</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml:591]: Please use "Cannot be deleted" or "Could not be deleted" instead. Do not include title here.

**Review by neslihanturan — CHANGES_REQUESTED** [review]
Hey @isdl03 , thanks for changes. I have added some minor suggestions. Please do not hesitate to say if you have doubts about my reviews, considering I am not a native speaker. 

Also, plase fix conflicts.

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (sivaraam) [reply app/src/main/res/values/strings.xml:86]: I'm not sure about whether it's legal or not (I'm not a native speaker, too 🙂). But I could certainly say that "Error logging in doesn't sound any better than "Login failed".

**Comment by neslihanturan** [comment]
Hey @isdl03 , how is the things going with this issue? Please do not hesitate to ask if you need further explanation. Currently minor reviews and conflict fixes should be done:)

**Comment by isdl03** [comment]
Currently our midsem exams are going so we will try to resolve the issue
after 25th February.

Thanks

**Comment by nicolas-raoul** [comment]
Would you mind solving the conflicts? :-)

**Comment by isdl03** [comment]
@nicolas-raoul I have made the changes as requested. Kindly review the same.

**Comment by nicolas-raoul** [comment]
Oops, seems that there is already another conflict... sorry but would you mind solving again? Thanks!

**Comment by nicolas-raoul** [comment]
You can either rebase on master, or just pull from master. Please let me know when you are done :-)

**Comment by isdl03** [comment]
I have resolved the conflicts.

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -73,35 +71,30 @@
...
   <string name="share_caption_hint">Caption</string>
   <string name="login_failed_network">Unable to login - network failure</string>
-  <string name="login_failed_wrong_credentials">Unable to login - please check your username and password</string>
   <string name="login_failed_throttled">Too many unsuccessful attempts. Please try again in a few minutes.</string>
   <string name="login_failed_blocked">Sorry, this user has been blocked on Commons</string>
   <string name="login_failed_2fa_needed">You must provide your two factor authentication code.</string>
   <string name="login_failed_generic">Login failed</string>
   <string name="share_upload_button">Upload</string>
   <string name="multiple_share_base_title">Name this set</string>
-  <string name="add_set_name_toast">Please provide a name for this set</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml:89]: Seems like some strings are removed in this PR and this is irrelevant with avoiding usage of "failure/success"

**Review by neslihanturan — CHANGES_REQUESTED** [review]
Some irrelevant changes exists

**Comment by neslihanturan** [comment]
hey @isdl03 , are you still interested?:)

**Pull request closed without merging**